src/common/components/Alert/Alert.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import styles from "./Alert.module.scss";
+
+export const Alert = ({ isVisible, children }) => {
+  const alertClassName = classNames(styles.alert, {
+    [styles["alert--isVisible"]]: isVisible
+  });
+
+  return (
+    <aside className={alertClassName}>{isVisible ? children : null}</aside>
+  );
+};
+
+Alert.propTypes = {
+  isVisible: PropTypes.bool
```

Comment:
`isVisible` should proibably be marked with `isRequired`, as the component needs it. There's no harm in adding `isRequired` even if there's a default value.